ultralib/include/macros.h

```diff
@@ -54,6 +54,6 @@
 // TODO create header for pokemon
 #define GET_POKEMON(obj) ((Pokemon*)((obj)->userData))
```

Comment:
oh we probably shouldn't have been putting these into the ultralib include dir. we can have our own include/macros.h and just include the ultralib one from there if we want.